glotaran/io/preprocessor/test/test_preprocessor.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+import xarray as xr
+
+from glotaran.io.preprocessor import PreProcessingPipeline
+
+
+def test_correct_baseline_value():
+    pl = PreProcessingPipeline()
+    pl.correct_baseline_value(1)
+    data = xr.DataArray([[1]])
+    result = pl.apply(data)
+    assert result == data - 1
+
+
+@pytest.mark.parametrize("indexer", (slice(0, 2), [0, 1]))
+def test_correct_baseline_average(indexer: slice | list[int]):
```

Comment:
Please also add a test for exclude and one that shows a combination of selection and exclude.

Reply:
We'll get back to it in a future PR ;)